pkg/api/converterfromv20180930preview.go

```diff
@@ -45,21 +45,25 @@ func ConvertFromV20180930preview(oc *v20180930preview.OpenShiftManagedCluster, o
 	return cs, nil
 }
 
+// mergeResourcePurchasePlan merges filled out fields from the external API to the internal representation, doesn't change nil fields.
 func mergeResourcePurchasePlan(oc *v20180930preview.OpenShiftManagedCluster, cs *OpenShiftManagedCluster) {
```

Comment:
Can you change it here to how we handle the resource plan conversion in pkg/api/converterfromadmin.go? Add an `if oc.Plan != nil` statement in L38, rename to `convertFromResourcePurchasePlanv20180930preview`, and passthrough cs.Plan as is if != nil. No reason to have such diff laying around.

Reply:
Moved nil check to ConvertFromV20180930preview func. 
Changed name to match convention, but left behaviour the same (doesn't change fields that have nil in external representation).